axonserver/src/main/java/io/axoniq/axonserver/grpc/GrpcExceptionBuilder.java

```diff
@@ -69,4 +70,18 @@ private static String cleanupDescription(ErrorCode standardErrorCode, String des
         }
         return createMessage(throwable);
     }
+
+    /**
+     * Checks if the given exception is a gRPC CANCELLED exception. This exception occurs when the client cancels the
+     * connection
+     * (usually on a non-clean exit)
+     *
+     * @param cause the exception to analyse
+     * @return true if the exception is that the connections is cancelled
+     */
+    public static boolean isCancelled(Throwable cause) {
```

Comment:
Not sure if that method is part of GrpcExceptionBuilder, seems more a utility class than a builder